api/src/handlers/items.rs

```diff
@@ -1,31 +1,44 @@
 use crate::db::models::{Item, ItemForm, NewItem};
 use crate::db::Connection as DbConnection;
 
-use actix_web::web::{HttpResponse, Json, Path};
+use actix_web::dev::HttpServiceFactory;
+use actix_web::web::{self, HttpResponse, Json, Path};
 use actix_web::Result;
 
-pub fn index(conn: DbConnection) -> Result<Json<Vec<Item>>> {
-    Item::select(&conn).map(|items| Json(items)).map_err(|e| {
+pub fn handler(path: &str) -> impl HttpServiceFactory {
```

Comment:
Shouldn't this go into the `handlers` mod.rs? Or do you recommend doing such a function for every handler module?